modules/data-streams/src/test/java/org/elasticsearch/datastreams/MetadataIndexTemplateServiceTests.java

```diff
@@ -251,7 +251,7 @@ private static DataStreamLifecycle.Downsampling randomDownsampling() {
     }
 
     private static DataStreamLifecycle.Downsampling randomNonEmptyDownsampling() {
-        var count = randomIntBetween(0, 10);
+        var count = randomIntBetween(0, 9);
```

Comment:
If I remember correctly this is inclusive right? In that case, setting this to 10 would allow it to reach 11 rounds, the `previous` one we always add plus 10. Right?